README.md

```diff
@@ -3,6 +3,18 @@
 [![Build Status](https://github.com/exasol/project-keeper/actions/workflows/ci-build.yml/badge.svg)](https://github.com/exasol/project-keeper/actions/workflows/ci-build.yml)
 Project Keeper Core: [![Maven Central &ndash; Project Keeper Core](https://img.shields.io/maven-central/v/com.exasol/project-keeper-core)](https://search.maven.org/artifact/com.exasol/project-keeper-core), Project Keeper Command Line Interface: [![Maven Central &ndash; Project Keeper Command Line Interface](https://img.shields.io/maven-central/v/com.exasol/project-keeper-cli)](https://search.maven.org/artifact/com.exasol/project-keeper-cli), Project Keeper Maven plugin: [![Maven Central &ndash; Project Keeper Maven plugin](https://img.shields.io/maven-central/v/com.exasol/project-keeper-maven-plugin)](https://search.maven.org/artifact/com.exasol/project-keeper-maven-plugin)
 
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Aproject-keeper-root&metric=alert_status)](https://sonarcloud.io/dashboard?id=com.exasol%3Aproject-keeper-root)
```

Comment:
Is this file changed due to new expectations / templates provided by pk?

Reply:
Yes, this is because of the restructuring in PK. I added `pom.xml`, so there are some changes.